uiowa/uiowa
# [ITS] Remove editor perms for Page content type

Resolves #7987 





# How to test


```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli
```

Masquerade as a user with only the Editor role
See that you cannot create Pages or edit Pages (such as `/alerts`)

Masquerade as a user with only the Publisher role
See that you can create and manage publishing workflow of Pages, but cannot edit their LB layout (such as `/alerts`)

Masquerade as a user with BOTH the Editor and Publisher role
See that you can do it all, including create new Pages, manage their publishing workflow, and edit new/existing Pages including editing their LB layout

Any other perms testing/considerations you can think of
